.github/scripts/make_debian.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd $GITHUB_WORKSPACE
```

Comment:
Unless $HOME an acceptable default for `cd` here, test for empty var?

Reply:
Since like 99% of GHA use this construction I'm reasonably confident it will exist in the context of GHA - if you're suggesting we ought generalize the script to make it work on non GHA environments?

Reply:
No need to change, then.  Just double-checking as nuances of GHA are still fairly new to me.  Thanks!